packages/fetchai/skills/weather_client/handlers.py

```diff
@@ -150,7 +150,8 @@ def _handle_propose(self, msg: FipaMessage, dialogue: Dialogue) -> None:
                 target=new_target_id,
                 performative=FipaMessage.Performative.ACCEPT,
             )
-            dialogue.outgoing_extend(accept_msg)
+            accept_msg.counterparty = msg.counterparty
+            dialogues.update(accept_msg)
```

Comment:
I'd prefer if we also allowed for `update` on the dialogue module

Reply:
Done

Reply:
Change to `dialogue.update(accept_msg)` so no need to import in line 125

Reply:
See above